packages/react-server-dom-webpack/src/ReactFlightWebpackNodeRegister.js

```diff
@@ -95,8 +97,6 @@ module.exports = function register() {
           return target.$$id;
         case '$$async':
           return target.$$async;
-        case 'name':
-          return target.name;
```

Comment:
This case was not covered by a unit test, but I think it's dead code and can not be reached.